packages/main/src/MenuItem.ts

```diff
@@ -264,6 +299,30 @@ class MenuItem extends ListItem implements IMenuItem {
 		});
 	}
 
+	get _list() {
+		return this.shadowRoot!.querySelector<List>("[ui5-list]")!;
```

Comment:
The list is conditionally rendered, so using `! `is not appropriate.

Reply:
Done